Biomass_speciesParameters.R

```diff
@@ -16,10 +16,9 @@ defineModule(sim, list(
                    before = c("Biomass_core")),
   reqdPkgs = list("crayon", "data.table", "disk.frame", "fpCompare", "ggplot2", "gridExtra",
                   "magrittr", "mgcv", "nlme", "purrr", "robustbase", "sf",
+                  "reproducible (>= 1.2.10.9001)", "SpaDES.core (>= 2.0.2.9004)",
```

Comment:
can we set these to v 2.1.0 or higher, which are/will be the current CRAN versions?

Reply:
(reproducible < v2.0.0 most likely won't work -- to many breaking changes going to v2)